internal/dic/index.go

```diff
@@ -75,6 +75,18 @@ func (idx IndexTable) CommonPrefixSearch(input string) (lens []int, ids [][]int)
 	return
 }
 
+// CommonPrefixSearch finds keywords sharing common prefix in an input
```

Comment:
__[gometalinter]__
* [golint] comment on exported method IndexTable.CommonPrefixSearchCallback should be of the form "CommonPrefixSearchCallback ..."